tests/e2e/performance/csi_tests/test_pod_reattachtime.py

```diff
@@ -232,12 +233,17 @@ def test_pod_reattach_time_performance(
             rsh_cmd = f"exec {pod_name} -- df -h {pod_path}"
             data_written = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
             logging.info(f"The amount of written data is {data_written}")
+            data_written_str = _ocp.exec_oc_cmd(rsh_cmd).split()[-4]
```

Comment:
why you run it twice ? lines 234 and 236 give the same results

Reply:
@Avilir fixed it. thank you for noticing!